bazaarvoice/emodb
# Datastorage reverting log back changes

Jira Issue
https://bazaarvoice.atlassian.net/browse/PD-228740

What Are We Doing Here?
Emodb is one of the 4 repositories where the critical vulnerabilities needs to be remediated in synk. Reverting log changes and Deleting of Java version local file from previous merge.

DOD:
All the remediation code should be tested, reviewed and merge to mainline.
Push to QA environment and validate.

Acceptance Criteria:
Java version file will be deleted from main branch.

How to Test and Verify

Check out local build :
Unit testing : mvn clean install
![image](https://github.com/bazaarvoice/emodb/assets/134580247/0dc22ac2-a1aa-4449-828c-104de875ed16)


Risk
Level : Low

Required Testing
Manual.

Risk Summary
Since risk level will be low,we will be monitoring for vulnerability issues of jackson library in snyk dashboard.
